setup/db_schema.sh

```diff
@@ -1,4 +1,6 @@
 #/bin/sh
 echo Upgrading db...
+DEBUG=True FLASK_APP=app.py flask db init
```

Comment:
Did you test this with already having a database file—Does it wipe it unexpectedly?

Reply:
great question, I didn't think of that; I'm glad you caught that

So to test:

1) I started with a fresh clone, ran the setup, and created a fresh DB.

2) created a new non-pre-populated Test Entry:

![db01](https://user-images.githubusercontent.com/73559781/152423750-fa9cea88-6d7e-46e6-a195-524a31959fec.JPG)

3) re-ran the setup script and made these relevant choices:

![db02](https://user-images.githubusercontent.com/73559781/152423892-f6bb9441-fdb9-4618-af1f-9cae6e92622c.JPG)

![db03](https://user-images.githubusercontent.com/73559781/152423911-9b4f138a-69f6-4eca-a8a6-ea8ac45c4e50.JPG)

![db04](https://user-images.githubusercontent.com/73559781/152423948-ecaaf3ea-bb71-429f-889b-8de5bcb5554a.JPG)

4) started the server back up and checked to see that Test Entry still existed:

![db05](https://user-images.githubusercontent.com/73559781/152424126-e1cb82bb-523a-4fd6-bcab-e334f4d43f2b.JPG)